package.json

```diff
@@ -111,9 +113,11 @@
     "sri-validate": "node build/sri-update --validate",
     "fmt": "prettier --write .",
     "prepare": "husky",
-    "prebuild": "node ./build/check-node-version.js",
+    "prebuild": "node ./build/check-node-version.js && npm run unpatch",
```

Comment:
One more thing to consider is that we could use the `prepare` script instead of a prebuild script. The reason we can't use postinstall is because that would trigger when someone uses `npm i axe-core`, but `prepare` only run when you call npm install in the axe-core repo. It would prevent having to run patch / unpatch over and over. @straker @dbjorge, do you see any reason not to do that?

Reply:
The main gotchas I'm aware of with `prepare` are that:

- It runs within a package consumer if the consumer uses a `"git:github.com/dequelabs/axe-core.git"` dependency on axe-core rather than a more typical npm dependency
- It's not treated consistently by all packaging tools; yarn v2+ ignores it, for example.

I don't think either of those matter enough to make me want to avoid it here; the former case is already broken if it isn't running our build process, and I don't think we need optimize for the latter (so long as there's a test that would catch the patch not applying as expected).